CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - `abi` directory to the `@cartesi/rollups` package, ideal for language bindings.
 
+### Changed
+
+- Changed inputs encodings to be blockchain agnostic
```

Comment:
Actually, this would go to 2.0.0, and the upcoming release is 1.1.0...
That's why I think we should use change sets, to have fewer problems while recording changes in each PR.

Reply:
Yeah. We could start using changeset